packages/next-auth/tests/getServerSession.test.ts

```diff
@@ -65,3 +66,58 @@ describe("Treat secret correctly", () => {
     expect(console.warn).toBeCalledTimes(1)
   })
 })
+
+describe("Return correct data", () => {
+  afterEach(() => {
+    jest.restoreAllMocks()
+  })
+
+  it("Should return null if there is no session", async () => {
+    const spy = jest.spyOn(core, "NextAuthHandler")
+    spy.mockReturnValue({ body: {} })
+
+    const session = await unstable_getServerSession(req, res, {
+      providers: [],
+      logger,
+      secret: "secret",
+    })
+
+    expect(session).toEqual(null)
+  })
+
+  it("Should return null if there is a config error", async () => {
```

Comment:
I think this test could be combined with "Error if missing NEXTAUTH_SECRET and secret"